chat-plugins/roomsettings.js

```diff
@@ -59,7 +59,10 @@ class RoomSettings {
 		return modchatOutput.join(' ');
 	}
 	modjoin() {
-		if (!this.user.can('makeroom') && !this.room.isPersonal) return this.button(this.room.modjoin ? this.room.modjoin : 'off', true);
+		if (!this.user.can('makeroom') && !this.room.isPersonal ||
+			!this.user.can('editroom', null, this.room)) {
```

Comment:
Well we have a *lot* of lines in the codebase that are >80 characters. Personally, I'd rather make the decision to split up a line on a case-by-case basis, depending on how dense and readable it is.